src/selectors/Entities/name.js

```diff
@@ -112,8 +112,9 @@ export const selectVaccinePatientHistory = patient => {
       ({ patientEventID, data }) =>
         patientEventID === vaccinationPatientEventID && validateJsonSchemaData(jsonSchema, data)
     )
-    .map(({ data: vaccinationNameNotes }) => ({
+    .map(({ id, data: vaccinationNameNotes }) => ({
```

Comment:
This is to stop an error when 2 doses of the same vaccine (item and batch) are locally dispensed in the same day (although likely an error, the error displayed was the generic invalid schema one and not that useful for users): 
`LOG  text {"name":"JSONForm","hostname":"react-native","level":40,"msg":"Invalid JSON Schema Error: schema is invalid: data/properties/causes/items/enum should NOT have duplicate items (items ## 2 and 3 are identical)`